packages/devtools_app/pubspec.yaml

```diff
@@ -48,8 +48,8 @@ dependencies:
     ^0.0.2
 #    path: ../../third_party/packages/primer_css
   split:
-    ^0.0.5
-#    path: ../../third_party/packages/split
+    # ^0.0.5
```

Comment:
I assume this will be removed before checkin.

Reply:
I will be revving the version numbers before checkin.